vmware_dvs/agent/dvs_neutron_agent.py

```diff
@@ -65,7 +65,6 @@ def __init__(self, vsphere_hostname, vsphere_login, vsphere_password,
             'agent_type': 'DVS agent',
             'start_flag': True}
 
-        self.setup_rpc()
         report_interval = cfg.CONF.AGENT.report_interval
```

Comment:
_report_state should be started after all rpc agents were initialized. So, please move L68-L72 under setup_rpc() function